### PR TITLE
[docs] Document Settings Improvements

### LINKED ISSUE #730: [docs] Document Settings
**UPDATE:**
https://sleipnirgroup.github.io/Choreo/document-settings/ 
* Update screenshot here

https://sleipnirgroup.github.io/Choreo/document-settings/robot-configuration/

(maybe wait on this one until diff drive works and the sketch configuration is implemented)
* Remove mention of Theoretical and Motor Calculator pages and update screenshot.

**REMOVE**
https://sleipnirgroup.github.io/Choreo/document-settings/file-export-settings/
* Entirely outdated.

**ADD**
Section describing the export config switch for including module forces.

### PR DESCRIPTION
**Pull request by DeltaDizzy**
Fixes #730

### CODE REVIEW

File: src/components/config/robotconfig/TheoreticalPanel.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ class RobotConfigPanel extends Component<Props, State> {
           <InputList noCheckbox rowGap={this.props.rowGap}>
             <Input
               title="Traction Accel Limit"
-              suffix={`${MetersOrFeet(imp)}/s`}
+              suffix={`${MetersOrFeet(imp)}/s^2`}
```
Comment (shueja) [inline src/components/config/robotconfig/TheoreticalPanel.tsx:47]:
```suggestion
              suffix={`${MetersOrFeet(imp)}/s²`}
```

File: docs/media/document-settings.png
Comment (shueja) [inline docs/media/document-settings.png:1]: Not sure we need this pic in addition to the one below: plus it doesn't have the ^2 fix.

**Review by shueja — CHANGES_REQUESTED** [review]

**Review by oh-yes-0-fps — APPROVED** [review]

File: docs/document-settings.md
Comment (shueja) [inline docs/document-settings.md:1]: With a single tab, this page is a bit of a stub, but I think we'll add more tabs for various features again in the future.

File: docs/document-settings/robot-configuration.md [hunk 2]
```diff
@@ -46,44 +44,17 @@ These values can be pre-filled using the [Motor Calculator](#motor-calculator) p
...
 ## Theoretical
 
 This panel displays calculated metrics about your robot, for reference and validation.
 
 ![robot-config-theoretical.png](../media/robot-config-theoretical.png)
 
+- **Traction Accel Limit**  $[m/s^2]$: The robot's maximum acceleration before wheels begin slipping
+- **Motor Accel Limit** $[m/s^2]$: The robot's maximum acceleration based on motor torque
 - **Floor Speed** $[m/s]$: The maximum speed reached by the robot when driving in a straight line and not rotating
 - **Floor Accel** $[m/s^2]$: The maximum acceleration reached by the robot when driving in a straight line and not rotating
```
Comment (shueja) [inline docs/document-settings/robot-configuration.md:58]:
```suggestion
- **Floor Accel** $[m/s^2]$: The maximum acceleration reached by the robot when driving in a straight line and not rotating. The minimum of traction and motor limits.
```

**Review by shueja — CHANGES_REQUESTED** [review]
Approved

File: docs/document-settings.md [hunk 3]
```diff
@@ -5,8 +5,6 @@ by clicking "Document Settings" in Choreo's main menu.
 
 ![Document Settings option in the main menu](./media/document-settings-menu.png)
 
-See the pages below for explanations of each tab in the Document Settings panel.
+See the page below for an explanations of how to use the Document Settings panel.
```
Comment (shueja) [inline docs/document-settings.md:8]:
```suggestion
See the page below for an explanation of how to use the Document Settings panel.
```

**Review by shueja — DISMISSED** [review]

**Pull request merged**